CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ Changes since v2.12
 - Fixed enabling a catalogue item after changing its form. (#2283)
 - Added missing decision text to pdf event list.
 - More compatible CSV reports. Line returns are removed from field values and CSV lines are separated with CRLF. (#2311)
+- Fixed editing a catalogue item. (#2322)
```

Comment:
Maybe "Fixed not all fields showing up when editing a catalogue item" ?

Reply:
It was completely broken, no fields showed up for me and I couldn't press save

Reply:
Ok in that case it's better like it is